db/models/users.py

```diff
@@ -15,6 +15,9 @@ class User(Base):
     last_name = Column(String, nullable=False)
     role_id = Column(Integer, ForeignKey("roles.id"), nullable=True)
     password = Column(String, nullable=False)
+    yoe = Column(Integer, nullable=False)
```

Comment:
what's this? I don't understand and I'm not sure it's "self explanatory" enough 🤔

Reply:
years of experience